docs/permissions/custom_policies.md

```diff
@@ -58,131 +60,243 @@ class MyPolicyProvider implements PolicyProviderInterface
+
+To customize the rendering, a value mapper eventually transforms the Limitation value and send it to a custom template.
+
+The value mapper implements [`Ibexa\AdminUi\Limitation\LimitationValueMapperInterface`](https://github.com/ibexa/admin-ui/blob/4.5/src/lib/Limitation/LimitationValueMapperInterface.php).
+
+Its `mapLimitationValue` function returns the Limitation value transformed for the needs of the template.
+
+``` php
+[[= include_file('code_samples/back_office/limitation/src/Security/Limitation/Mapper/CustomLimitationValueMapper.php') =]]
+```
+
+Then register the service with the `ibexa.admin_ui.limitation.mapper.value` tag and set the `limitationType` attribute to Limitation type's identifier:
+
+``` yaml
+[[= include_file('code_samples/back_office/limitation/config/append_to_services.yaml', 10, 13) =]]
 ```
 
-If you want to completely override the way of rendering custom Limitation values in the role view,
-create a Twig template containing block definition which follows the naming convention:
-`ez_limitation_<LIMITATION TYPE>_value`. For example:
+When a value mapper exists for a Limitation, the rendering uses a Twig block named `ez_limitation_<lower_case_identifier>_value` where `<lower_case_identifier>` is the Limitation identifier in lower case.
+In this example, block name is `ez_limitation_customlimitation_value` as the identifier is `CustomLimitation`.
+
+This template receive a `values` variable which is the return of the `mapLimitationValue` function from the corresponding value mapper.
 
 ``` html+twig
-{# This file contains block definition which is used to render custom Limitation values #}
-{% block ez_limitation_custom_value %}
-    <span style="color: red">{{ values }}</span>
-{% endblock %}
+[[= include_file('code_samples/back_office/limitation/templates/themes/standard/limitation/custom_limitation_value.html.twig') =]]
 ```
 
-Add it to the configuration under `ibexa.system.<SCOPE>.limitation_value_templates`:
+To have your block found, you have to register its template. Add the template to the configuration under `ezplatform.system.<SCOPE>.limitation_value_templates`:
+
+``` yaml
+[[= include_file('code_samples/back_office/limitation/config/packages/ibexa_security.yaml') =]]
+```
 
-```yaml
-ibexa:
-    system:
-        default:
-            limitation_value_templates:
-                - { template: limitation/custom_limitation_value.html.twig, priority: 0 }
+Provide translations for your custom limitation form in the `ezplatform_content_forms_policies` domain. For example, `translations/ezplatform_content_forms_policies.en.yaml`:
 
+``` yaml
+[[= include_file('code_samples/back_office/limitation/translations/ezplatform_content_forms_policies.en.yaml') =]]
 ```
 
-!!! note
+### Checking user custom Limitation
```

Comment:
Consider rewriting all headers to remove the gerund.

Reply:
Done in ea5117c9d97093cfafd42d56ba04a7cbb204c2f1